packages/shared-types/src/ComponentThemeVariables.ts

```diff
@@ -519,6 +519,7 @@ export type ColorPickerTheme = {
   popoverFooterPadding: Spacing['xSmall']
   popoverFooterTopBorderWidth: Border['widthSmall']
   colorMixerButtonContainerLeftMargin: Spacing['xSmall']
+  spacing: any
```

Comment:
Can you type this?

Reply:
This is very tricky to type, almost impossible if I'm correct.